snapcraft/commands/remote.py

```diff
@@ -69,7 +69,13 @@ class RemoteBuildCommand(BaseCommand):
         option, followed by the build number informed when the remote
         build was originally dispatched. The current state of the
         remote build for each architecture can be checked using the
-        --status option."""
+        --status option.
+
+        To set a timeout on the remote-build command, use the option
+        ``--launchpad-timeout=<seconds>``. The timeout is local, so the build on
+        launchpad will continue even if the local instance of snapcraft is
+        interrupted or times out.
```

Comment:
This might be a good thing to include when exiting due to either a timeout or a keyboard interrupt. I'm imagining some output like:

```
Remote build command timed out. Build is still running on launchpad and may be reviewed at https://launchpad.net/~user/blah/blah
```

Reply:
Good idea.  I added details on how to recover the build via the command line rather than posting the URL (which is significantly more complicated, mostly because each architecture in a build has its own URL and you have to query launchpad for the urls).

Reply:
Even better! Thanks!